rubygem/lib/zeus/m.rb

```diff
@@ -192,6 +196,18 @@ def execute
 
         test_arguments = build_test_arguments
 
+        if @reporter
+          begin
+            require 'minitest/reporters'
+            klass_name = @reporter.capitalize + 'Reporter'
```

Comment:
It seems a bit unpleasant to assume that the class names will always be in this format. Can we just do `--reporter SpecReporter` instead?